app/javascript/views/Login.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { inject } from 'mobx-react';
+import SessionStore from '../stores/session_store';
+import { Header, Form } from 'semantic-ui-react';
+import { RouterStore } from 'mobx-react-router';
+import DbSqlIcon from '../shared/DbSqlIcon';
+
+interface InjectedProps {
+  sessionStore: SessionStore;
+  routerStore: RouterStore;
+}
+
+@inject('sessionStore', 'routerStore')
+export default class Login extends React.Component {
+  private email: string = '';
+  private password: string = '';
+  get injected() {
+    return this.props as InjectedProps;
+  }
+
+  login() {
+    this.injected.sessionStore.login(
+      this.email,
+      this.password
```

Comment:
would it make sense to clear the state of the `password` after performing the login (some goes for the email)?

Reply:
then i should probably clear the input field too?